### PR TITLE
Add policy for provisioning cisagov/cyhy-tf-root

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR adds a policy that allows the provisioning of the resources in [cisagov/cyhy-tf-root](https://github.com/cisagov/cyhy-tf-root).

## 💭 Motivation and context ##

Without this policy, we won't be able to apply the Terraform in [cisagov/cyhy-tf-root](https://github.com/cisagov/cyhy-tf-root).

## 🧪 Testing ##

After I created this policy, I verified that I was able to successfully `terraform apply` and `terraform destroy` all of the resources for https://github.com/cisagov/cyhy-tf-root/pull/12.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dv4harr10 — APPROVED** [review]

**Review by jsf9k — CHANGES_REQUESTED** [review]
Don't you need to update the `terraform-docs` output in `README.md` as well?

**Comment by dav3r** [comment]
> Don't you need to update the `terraform-docs` output in `README.md` as well?

Thanks for the reminder!  Please see https://github.com/cisagov/cool-accounts-cyhy-tf-root/pull/3/commits/5cbfa41c6e30c11303d208b3f8af9a0aff1afeac.

**Review by jsf9k — APPROVED** [review]

**Comment by dav3r** [comment]
@mcdonnnj - Note https://github.com/cisagov/cool-accounts-cyhy-tf-root/pull/3/commits/0f9c2cfd77e0d72675f8b61894154d0e6fabd9c4 per our discussion yesterday.

**Review by jsf9k — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]
Approved even harder!

File: variables.tf [hunk 1]
```diff
@@ -50,6 +50,18 @@ variable "provisionaccount_role_name" {
   type        = string
 }
 
+variable "provisioncyhyroot_policy_description" {
+  default     = "Allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
+  description = "The description to associate with the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
```
Comment (jsf9k) [inline variables.tf:55]:
```suggestion
  description = "The description to associate with the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
  nullable    = false
```

File: variables.tf [hunk 2]
```diff
@@ -50,6 +50,18 @@ variable "provisionaccount_role_name" {
...
+  type        = string
+}
+
+variable "provisioncyhyroot_policy_name" {
+  default     = "ProvisionCyHyRoot"
+  description = "The name to assign the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
```
Comment (jsf9k) [inline variables.tf:62]:
```suggestion
  description = "The name to assign the IAM policy that allows sufficient permissions to provision all AWS resources required by cisagov/cyhy-tf-root."
  nullable    = false
```

**Review by jsf9k — CHANGES_REQUESTED** [review]

**Review by jsf9k — APPROVED** [review]

**Review by felddy — APPROVED** [review]
💎Approving harder still! 💎 
pingu-smack

**Comment by jsf9k** [comment]
> 💎Approving harder still! 💎 
> pingu-smack
> 

Pingu!

**Pull request merged**